### PR TITLE
feat(frontend): leave BBB table

### LINKED ISSUE #706: 🚀 [Feature] Frontend: Leave BBB room
## 🚀 Feature

We want the frontend to register when the User leaves BigBlueButton table.
:warning: This has been postponed due to several infrastructure issue, which have to be addressed first.

In the backend a redirect url when a meeting is ended to (eg. https://stage1.app.dreammall.earth/ or a new page)

## 🤖 ToDo

- [x] Redirect to home

### PR DESCRIPTION
**Pull request by Bettelstab**
## 🍰 Pullrequest
When the URL of the EmbeddedRoom iframe changes to `/` of the current origin, like the home page of our BBB instance, the iframe is removed and a redirect to the home page of the app is triggered.

### Issues
- fixes #706 

### Todo
- [x] Tests

### CODE REVIEW

File: frontend/src/components/embedded-room/EmbeddedRoom.vue [hunk 1]
```diff
@@ -1,18 +1,34 @@
...
+const isRedirected = ref(false)
+
+const emit = defineEmits(['closed'])
+
+const onRedirect = (event: Event): void => {
+  const src = (event.target as HTMLIFrameElement).src
+  if (src === new URL(src).origin) {
+    // We were redirected to the home page of our BigBlueButton server, hence we left the meeting
+    isRedirected.value = true
+    emit('closed')
```
Comment (mahula) [inline frontend/src/components/embedded-room/EmbeddedRoom.vue:31]: Please test the emit.

**Review by mahula — CHANGES_REQUESTED** [review]

File: frontend/src/components/embedded-room/EmbeddedRoom.vue (hunk 1 above)
Comment (Bettelstab) [reply frontend/src/components/embedded-room/EmbeddedRoom.vue:31]: Of course. I am still working on the tests.

**Comment by Mogge** [comment]
@Bettelstab Please update the branch. The tests changed a lot

**Review by Mogge — CHANGES_REQUESTED** [review]
It does not work for me. The redirect is not triggered. The event `onRedirect` is never fired.

**Comment by Bettelstab** [comment]
> It does not work for me. The redirect is not triggered. The event `onRedirect` is never fired.

I forgot to mention that to test this code locally, you have to change the iframe src by hand. (Or edit the iframe content and add a link.) Did you try something like that? 
Possibly I could come up with a better story to show this behavior...

**Review by Elweyn — CHANGES_REQUESTED** [review]
Tried it on stage1 and it does not work.
After clicking on konferenz close I get 
Screenshot 2024-06-27 at 12 33 40
After clicking on konferenz leave I get the same screen

**Comment by Bettelstab** [comment]
=> Paused until we have subscriptions (#727)

**Pull request closed without merging**